### PR TITLE
DASH-106. Order processing job.

### PR DESCRIPTION
**Pull request by andreyvEze**

### CODE REVIEW

File: server/src/external/fio.mjs [hunk 1]
```diff
@@ -74,6 +100,20 @@ class Fio {
       .toNumber();
   }
 
+  convertUsdcToFio(amount, roe) {
+    return new MathOp(amount)
```
Comment (trukhilio) [inline server/src/external/fio.mjs:108]: I think we should add `roe == null` check here

**Review by trukhilio — APPROVED** [review]

**Comment by trukhilio** [comment]
Check comments

File: server/tests/use-case/webhook/coinpayments-webhook.mjs [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    tax: 0,
+    txn_id: txnId,
+  };
+
+  if (net) body.net = net;
+
+  const res = await api.request({
+    headers: {
+      connection: 'close',
+      // 'content-length': '0',
```
Comment (trukhilio) [inline server/tests/use-case/webhook/coinpayments-webhook.mjs:137]: Please, remove or left comment why we need it here

File: server/tests/use-case/webhook/coinpayments-webhook.mjs [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+      host: '',
+    },
+    url: '/payments/webhook/',
+    method: 'post',
+    body,
+  });
+
+  if (res.status === 0 && res.error) {
+    // eslint-disable-next-line no-console
+    console.log(res);
```
Comment (trukhilio) [inline server/tests/use-case/webhook/coinpayments-webhook.mjs:151]: Remove or left comment

**Review by trukhilio — COMMENTED** [review]
Check comments

File: server/src/models/OrderItem.mjs [hunk 4]
```diff
@@ -108,7 +108,7 @@ export class OrderItem extends Base {
         ? 'o."refProfileId" IS NULL'
         : `o."refProfileId" = ${refProfileId}`;
 
-    const [orderItemAmount] = await Sequelize.query(`
+    const [orderItemAmount] = await OrderItem.sequelize.query(`
```
Comment (NikolasStelmakh) [inline server/src/models/OrderItem.mjs:111]: what purpose of setting Sequelize as init parameter and using like this?

File: server/src/external/fio.mjs [hunk 5]
```diff
@@ -74,6 +104,20 @@ class Fio {
...
+      .div(roe)
+      .round(9, 2)
+      .toNumber();
+  }
+
+  logError(e) {
+    if (e.errorCode !== 404) logger.error(e.message);
+    if (e.json) {
+      logger.error(e.json);
+    }
```
[the remark this answers is not in the document]
Comment (andreyvEze) [reply server/src/external/fio.mjs:118]: there should not be else case, all errors beside 404 will log on 115 line. json in error is optional and we want to see what is there if set

File: server/tests/use-case/webhook/coinpayments-webhook.mjs (hunk 2 above)
Comment (andreyvEze) [reply server/tests/use-case/webhook/coinpayments-webhook.mjs]: removed

File: server/tests/use-case/webhook/coinpayments-webhook.mjs (hunk 3 above)
Comment (andreyvEze) [reply server/tests/use-case/webhook/coinpayments-webhook.mjs:151]: left comment

File: server/src/external/fio.mjs (hunk 1 above)
Comment (andreyvEze) [reply server/src/external/fio.mjs:108]: done

**Review by trukhilio — APPROVED** [review]

**Pull request merged**